tests/specs/init.js

```diff
@@ -197,3 +197,158 @@ test('hoodie.store initialization without session', function (t) {
   var storeAjaxParam = CustomStoreMock.lastCall.args[1]
   t.is(storeAjaxParam.ajax(), undefined, 'no authorization header without session')
 })
+
+test('"hoodie.store.push" is called on "pre:signout"', function (t) {
+  t.plan(2)
+
+  var hoodie = {
+    account: {
+      id: 0,
+      on: simple.stub(),
+      isSignedIn: simple.stub()
+    },
+    store: {
+      push: function () {
+        t.pass('store.push called on "signout"')
+      }
+    },
+    connectionStatus: {
+      on: simple.stub()
+    }
+  }
+
+  init(hoodie)
+  var signOutHandler = findEventHandler(hoodie.account.on.calls, 'pre:signout')
+  var hooks = []
+  signOutHandler({hooks: hooks})
+  t.is(hooks.length, 1, 'one pre:signout hook registered')
+  hooks[0]()
+})
+
+test('"hoodie.store.*" is *not* called when "hoodie.account.isSignedIn()" returns "false"', function (t) {
+  t.plan(1)
+
+  var hoodie = {
+    account: {
+      id: 0,
+      on: simple.stub(),
+      isSignedIn: simple.stub().returnWith(false)
+    },
+    store: simple.stub(),
```

Comment:
the method we want to stub here is not `hoodie.store()` but `hoodie.store.connect()`. Sorry I missed this before, could you please change it?